### PR TITLE
Update instruction on how to allow mic access.

### LINKED ISSUE #349: Old info in Readme on how to allow mic access.
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**

Instruction number 6 should be changed, as to allow mic access, we need to go to options which is accessed through `right click on extension icon` and then clicking `Allow` button.

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by rupav**
Fixes #349 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

- Update Readme instruction on how to allow mic access.
-
-

### CODE REVIEW

**Comment by rupav** [comment]
@gabru-md @stealthanthrax @zamhaq please review.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ $ cd src
 3. Turn on developer mode if not already on.
 4. Load unpacked extension from `src` folder.
 5. Click on the SUSI icon you see on your browser toolbar (or) Use the keybkeyboard shortcut ``Alt+Shift+s`` to open. 
-6. Click on the settings icon (top left of the popup) to allow mic access.
+6. Right click on the manifest icon (susi ai icon on chrome), then click on `Options`. `Allow`  mic-access from there.
```
Comment (gabru-md) [inline README.md]: change `susi ai` to `SUSI.AI`

**Comment by rupav** [comment]
Done @gabru-md . Let me know if I need to squash. I did these changes through github UI only (without using git), so if its possible please squash from your end. Thanks 😃

**Comment by gabru-md** [comment]
no problem @rupav I'll just squash and merge the commits that you made. :D

**Pull request merged**